avocado/etc/avocado/avocado.conf

```diff
@@ -37,6 +37,9 @@ profilers = etc/avocado/sysinfo/profilers
 [runner.output]
 # Whether to display colored output in terminals that support it
 colored = True
+# Whether to force colored output to non-tty outputs (e.g. log files)
+# Allowed values: auto, always, never
```

Comment:
This is kind of a "white lie", in the sense that any other value other than "auto" and "never" will force colored output, right?  We should either change the documentation or change the logic to match the comment.

Reply:
All good points, I simply copied typical command line switches from `grep`, `git diff`, etc. which are usually set to "always", "never", or "auto". I was actually wondering whether it would make sense to reuse the `colored` setting as there are some intersections but I guess preserving backwards compatibility is of higher priority. Will fix the logic in a jiffy.